src/main/java/jenkins/plugins/logstash/LogstashConfiguration.java

```diff
@@ -180,6 +185,21 @@ public boolean configure(StaplerRequest staplerRequest, JSONObject json) throws
     {
       activeIndexer = logstashIndexer;
     }
+    
+    try {
+      // Validate mimetype and throw exception in case found invalid.
+      if (logstashIndexer instanceof ElasticSearch) {
```

Comment:
Why not just declare a noop `validate()` method on  `LogstashIndexer` and override it in `ElasticSearch` implementation?